bench/wtperf/runners/evict-btree-readonly.wtperf

```diff
@@ -0,0 +1,12 @@
+# wtperf options file: evict btree configuration
```

Comment:
I would say it is both.  I specifically chose eviction to make sure there are no surprising interactions when the cache fills up.  But I also want to make sure that readonly mode does not impact performance (potentially it could be a slight improvement if we run fewer threads or internal threads do less work, I guess).